src/main/webapp/app/exercises/shared/structured-grading-criterion/grading-instructions-details/grading-instructions-details.component.html

```diff
@@ -1,40 +1,107 @@
-    #markdownEditor
-    style="height: 350px"
-    class="markdown-editor"
-    [markdown]="questionEditorText"
-    [showPreviewButton]="true"
-    [domainCommands]="domainCommands"
-    (markdownChange)="prepareForSave()"
-    (textWithDomainCommandsFound)="domainCommandsFound($event)"
->
-    <!-- Preview -->
-    <ng-container #preview>
-        <div class="pt-1 background-editor-high markdown-preview">
-            <div *ngFor="let criterion of exercise.gradingCriteria">
-                <label class="font-weight-bold" [innerHTML]="criterion!.title! | htmlForMarkdown"></label>
-                <table class="table table-bordered">
-                    <thead>
-                        <tr>
-                            <td class="font-weight-bold">Credits</td>
-                            <td class="font-weight-bold" jhiTranslate="artemisApp.exercise.gradingScale">Grading Scale</td>
-                            <td class="font-weight-bold" jhiTranslate="artemisApp.exercise.instrDescription">Instruction Description</td>
-                            <td class="font-weight-bold">Feedback</td>
-                            <td class="font-weight-bold" jhiTranslate="artemisApp.exercise.usageCount">Max Count in Score</td>
-                        </tr>
-                    </thead>
-                    <tbody>
-                        <tr *ngFor="let instruction of criterion.structuredGradingInstructions">
-                            <td style="width: 5%" [innerHTML]="instruction.credits"></td>
-                            <td style="width: 10%" [innerHTML]="instruction!.gradingScale! | htmlForMarkdown"></td>
-                            <td style="width: 40%" [innerHTML]="instruction!.instructionDescription! | htmlForMarkdown"></td>
-                            <td style="width: 35%" [innerHTML]="instruction!.feedback! | htmlForMarkdown"></td>
-                            <td style="width: 10%" [innerHTML]="instruction.usageCount"></td>
-                        </tr>
-                    </tbody>
-                </table>
+<div *ngIf="!exercise.gradingInstructionFeedbackUsed">
+    <jhi-markdown-editor
+        #markdownEditor
+        style="height: 350px"
+        class="markdown-editor"
+        [markdown]="questionEditorText"
+        [showPreviewButton]="true"
+        [domainCommands]="domainCommands"
+        (markdownChange)="prepareForSave()"
+        (textWithDomainCommandsFound)="domainCommandsFound($event)"
+    >
+        <!-- Preview -->
+        <ng-container #preview>
+            <div class="pt-1 background-editor-high markdown-preview">
+                <div *ngFor="let criterion of exercise.gradingCriteria">
+                    <label class="font-weight-bold" [innerHTML]="criterion!.title! | htmlForMarkdown"></label>
+                    <table class="table table-bordered">
+                        <thead>
+                            <tr>
+                                <td class="font-weight-bold">Credits</td>
```

Comment:
I would expect this to be translated as well

Reply:
oh well, I did not focus on the existing structure, you're right, this one should be translated 😉

Reply:
I guess `credits ` and `feedback ` are same for English and German, that's why they are not included in the translation files at first. But I am a beginner when it comes to German 😄

Reply:
I see. I would've translated it with "Punkte", but now I realise credits are referred to as points on other places in Artemis (most notably the points selector in the exercise creation) and I wonder why they are referred to as credits here (but are still the same thing from my understanding). I agree this is out of scope for this PR though.